app/screens/image-gallery-viewer/gallery-image.tsx

```diff
@@ -43,13 +47,31 @@ const MAX_SCALE = 6
 const SWIPE_UP_THRESHOLD = 10
 const SWIPE_TO_CLOSE_THRESHOLD = 100
 
+import * as MediaLibrary from "expo-media-library"
+import type { Asset as Media } from "expo-media-library"
+
+const getUserAssets = async (): Promise<Media[] | null> => {
+  const { assets } = await MediaLibrary.getAssetsAsync({
+    mediaType: ["photo", "video", "audio", "unknown"],
+  })
+  if (!assets) {
+    return null
+  }
+  return assets
+}
+
+const getMediaInfo = async (asset: Asset): Promise<MediaLibrary.AssetInfo> => {
```

Comment:
@emadbaqeri Can you please define this function (getMediaInfo) in the asset-service.ts?